### PR TITLE
feat: release job definition

### PR DESCRIPTION
**Pull request by kuisathaverat**
* Create a new folder Ingest-management for the jobs related
* Create a new job to run the release (only definition)
* Remove the weekly repository scan trigger

### CODE REVIEW

File: .ci/jobs/folders.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (v1v) [inline .ci/jobs/folders.yml:1]: NIT:
rename file with `ingest-management.yml`?

Comment (mdelapenya) [reply .ci/jobs/folders.yml:1]: I'd suggest using `ingest-manager` everywhere

File: .ci/jobs/folders.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+#https://docs.openstack.org/infra/jenkins-job-builder/project_folder.html
+- job:
+    name: Ingest-management
```
Comment (mdelapenya) [inline .ci/jobs/folders.yml:4]: Ditto

Comment (mdelapenya) [inline .ci/jobs/folders.yml:4]:
```suggestion
    name: Ingest-manager
```

File: .ci/jobs/folders.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    description: Ingest management related Jobs
```
Comment (mdelapenya) [inline .ci/jobs/folders.yml:5]:
```suggestion
    description: Ingest manager related Jobs
```

File: .ci/jobs/folders.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    project-type: folder
+
+- view:
+    name: Ingest-management
```
Comment (mdelapenya) [inline .ci/jobs/folders.yml:9]:
```suggestion
    name: Ingest-manager
```

File: .ci/jobs/release.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+- job:
+    name: Ingest-management/release-distribution
```
Comment (mdelapenya) [inline .ci/jobs/release.yml:3]:
```suggestion
    name: Ingest-manager/release-distribution
```

File: .ci/jobs/release.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    display-name: Release Package Distribution
+    description: Jenkins pipeline to release Docker images to an environment.
+    view: Ingest-management
```
Comment (mdelapenya) [inline .ci/jobs/release.yml:6]:
```suggestion
    view: Ingest-manager
```

**Comment by mtojek** [comment]
@ruflin this is relatively old PR. Can we close this one or is there a bigger initiative behind?

**Comment by ruflin** [comment]
@kuisathaverat Can you comment on this? The idea was to get deployments of the registry automated but I need @kuisathaverat to comment on where we are here at the moment.

**Comment by kuisathaverat** [comment]
This is the project definition, so it only creates the job in Jenkins no logic, the other PR is the one that has the logic https://github.com/elastic/package-storage/pull/165 I changing it to make a rollout instead of an update of the helm values that was the first approach

**Comment by ruflin** [comment]
I'm confused. In which order should I look at the 2 PR's? Which one needs to go in first?

**Comment by kuisathaverat** [comment]
> I'm confused. In which order should I look at the 2 PR's? Which one needs to go in first?

does not matter, one is nothing with the other. This one does not have too much to check is a Job in Jenkins that's it. The other is the real pipeline.

**Pull request merged**